src/Resources/contao/library/Contao/Template.php

```diff
@@ -265,7 +265,7 @@ public function output()
 
 		header('Vary: User-Agent', false);
 		header('Content-Type: ' . $this->strContentType . '; charset=' . \Config::get('characterSet'));
-
+		 // FIXME: would be cool to throw a response exception here but sadly the hook is in the way.
```

Comment:
So what's the point of the FIXME then? Do you want to talk about removing the hook? Or do you just want to mention it; in this case, you should remove the `FIXME:` and leave the comment only.